plugin.go

```diff
@@ -24,12 +24,14 @@ import (
 	"go.uber.org/zap"
 
 	// gzip grpc compressor
+	"google.golang.org/grpc"
 	_ "google.golang.org/grpc/encoding/gzip"
 )
 
 const (
-	name             string = "otel"
-	configurationKey string = "http.otel"
+	name                 string = "otel"
+	httpConfigurationKey string = "http.otel"
+	grpcConfigurationKey string = "grpc.otel"
```

Comment:
According to my comment, you can use the `otel` top key configuration and use it only in 1 place. So, no need to have a duplicated config under the `grpc` or `http` sections.

Reply:
Do you want to break backward compatibility promise here?

Reply:
I suggest to do it in separate PR to have easier way to track changes. Ofc I can do it

Reply:
@rustatian anyway done 😄

Reply:
> Do you want to break backward compatibility promise here?

We have a major release update :) `v2023`. But, we also going to update the configuration to automatically handle this case when you're using the old config.

Reply:
`otel` middleware won't use a real configuration btw 😃 It receives it from the `config` plugin. Which will store internally, also the `http.otel` key. So, the new RR would be able to transform the old config and store internally 2 aliases: `http.otel` and `otel`.

Reply:
Thus, the new RR would not have BCs ⚡ Instead, we would have the ability to significantly reduce the `if .. else` noise in other plugins.